src/memote/suite/cli/runner.py

```diff
@@ -910,34 +615,13 @@ def online(note, github_repository, github_username):
             "the current branch's commit history."
         )
         sys.exit(1)
-    if note == "memote-ci access":
-        note = "{} to {}".format(note, github_repository)
-
-    # Github API calls
-    # Set up the git repository on GitHub via API v3.
-    gh_repo_name, auth_token, repo_access_token = _setup_gh_repo(
-        github_repository, github_username, note
-    )
-
-    # Travis API calls
-    # Configure Travis CI to use Github auth token then return encrypted token.
-    secret = _setup_travis_ci(gh_repo_name, auth_token, repo_access_token)
-
-    # Save the encrypted token in the travis config then commit and push
-    LOGGER.info("Storing GitHub token in '.travis.yml'.")
-    config = te.load_travis_configuration(".travis.yml")
-    global_env = config.setdefault("env", {}).get("global")
-    if global_env is None:
-        config["env"]["global"] = global_env = {}
-    try:
-        global_env["secure"] = secret
-    except TypeError:
-        global_env.append({"secure": secret})
-    te.dump_travis_configuration(config, ".travis.yml")
-    LOGGER.info("Add, commit and push changes to '.travis.yml' to GitHub.")
-    repo.index.add([".travis.yml"])
-    repo.git.commit("--message", "chore: add encrypted GitHub access token")
-    repo.git.push("--set-upstream", "origin", repo.active_branch.name)
+    LOGGER.info("Push changes to GitHub.")
+    _setup_gh_repo(github_token, github_repository, github_username)
+    active_branch = repo.active_branch.name
+    repo.git.push("--set-upstream", "origin", active_branch)
+    repo.heads[deployment].checkout()
```

Comment:
This will fail during CI because the memote test repo does not include a deployment branch.

The deployment branch (default, "gh-pages") gets created on the post-gen-project hook of cookiecutter-memote. To me, it is unclear whether I should

1. modify the test repo;
2. modify `online` to create the deployment branch if it does not exist locally; or
3. push all local branches blindly.

Reply:
I opted for (2) to make tests pass for now.